simplq/src/components/common/QrScanner/QrScanner.jsx

```diff
@@ -11,10 +11,10 @@ export default () => {
 
   const handleScan = (data) => {
     if (data != null) {
-      const res = this.getRoute(window.location.origin, data.text);
+      const res = getRoute(window.location.origin, data.text);
```

Comment:
This file is a problem, right? Where are props and getRoute coming from?

Reply:
Good callout.

Reply:
`getRoute` has an issue in jsx, also in the master branch. 
I'm trying to guess what this method should do and where is it defined

Reply:
Yeah, this looks like a miss during a previous refactor. You can skip changing this file for now till we figure out how to handle it better. So can you revert changes to this file and just add changes for the other one?

Reply:
Ok, I've revert changes of this file changing only the extension :wink: